bindings/python/src/OpenSpaceToolkitPhysicsPy/Time/Interval.cpp

```diff
@@ -231,10 +231,10 @@ inline void OpenSpaceToolkitPhysicsPy_Time_Interval(pybind11::module& aModule)
                 Generate a grid of instants with a given time step.
 
                 Args:
-                    aTimeStep (Duration): Time step.
```

Comment:
This should be `time_step`

Reply:
discusssed that this is because there is no kwarg here